spec/controllers/revisions_controller_spec.rb

```diff
@@ -3,9 +3,10 @@
 
 describe RevisionsController do
   describe '#index' do
```

Comment:
Add a comment that we're using this spec with different course types to check the behavior of course start/end times and how these interact with the Course.revisions scope,